bin/s2nc.c

```diff
@@ -236,6 +240,10 @@ int main(int argc, char *const *argv)
     int echo_input = 0;
     int use_corked_io = 0;
     int use_tls13 = 0;
+    int keyshares_count = 0;
+    uint16_t keyshares[S2N_ECC_EVP_SUPPORTED_CURVES_COUNT] = { UINT16_MAX };
```

Comment:
Since `S2N_ECC_EVP_SUPPORTED_CURVES_COUNT` is not in the public API, could this be redefined in common.h? Or included in the public API?

My main concern is if we need to include it in our demo app, then our customers may need it as well. We shouldn't force them to use a value which we don't support.

Reply:
Fair point, we may need to expose this value through common.h.

Reply:
Let's not expose any new values until the interface is finalized and we know we need them.